clj-http-lite/src/martian/clj_http_lite.clj

```diff
@@ -0,0 +1,31 @@
+(ns martian.clj-http-lite
+  (:require [clj-http.lite.client :as http]
+            [cheshire.core :as json]
+            [martian.core :as martian]
+            [martian.interceptors :as interceptors]))
+
+(defn- prepare-response-headers [headers]
+  (reduce (fn [m [k v]] (assoc m (keyword k) v)) {} headers))
+
+(def perform-request
+  {:name ::perform-request
+   :leave (fn [{:keys [request] :as ctx}]
+            (assoc ctx :response (-> (http/request request)
+                                     ;; convert keys to keywords
+                                     (update-in [:headers] prepare-response-headers))))})
+
+(def default-interceptors
+  (concat martian/default-interceptors [interceptors/default-encode-body interceptors/default-coerce-response perform-request]))
+
+(defn bootstrap [api-root concise-handlers & [opts]]
+  (martian/bootstrap api-root concise-handlers (merge {:interceptors default-interceptors} opts)))
+
+(defn bootstrap-swagger [url & [{:keys [interceptors] :as params}]]
+  ;; :json not supported by clj-htt-lite
```

Comment:
Hi,
This comment confuses me, it says :json is not supported but the next line uses it anyway?

Reply:
`clj-http-lite` does not support `:as :json` (yet) is what I am trying to say. With swagger, we have to have json in `swagger-definition` so we create the thing right there in `clj_http_lite.clj` threading `:body` through `(json/parse-string keyword)`. ;)

Reply:
Thanks for the clarification. I think it would be better in that case to remove the `:as :json` and the comment. It can be delegated to clj-http-lite in the future when it supports it.

Reply:
You are right about the confusion. Removed the misleading map and massaged the comment.